index.js

```diff
@@ -248,6 +245,30 @@ Peer.prototype.send = function (chunk) {
   self._channel.send(chunk)
 }
 
+/**
+ * Add a Transceiver to the connection.
+ * @param {String} kind
+ * @param {Object} init
+ */
+Peer.prototype.addTransceiver = function (kind, init) {
```

Comment:
So this is now called internally, can't we rename it to `_addTransceiver` as private implementation design?
Or if there is still a use case to call it explicitly, it should be documented in readme.

Reply:
I think there's still a use-case. If you already know exactly how many transceivers you will need, there's no point in renegotiating multiple times. You could add all the transceivers you'll need right away.

Agreed it should be documented.